snappy/info_test.go

```diff
@@ -80,8 +79,8 @@ type: os`)
 	}
 
 	// now remove the channel
-	storeMinimalRemoteManifest("app."+testDeveloper, "app", testDeveloper, "1.10", "Hello.", "")
-	storeMinimalRemoteManifest("fwk", "fwk", testDeveloper, "1.0", "Hello.", "")
+	storeMinimalRemoteManifest("app", testDeveloper, "1.10", "Hello.", "")
+	storeMinimalRemoteManifest("fwk", testDeveloper, "1.0", "Hello.", "")
```

Comment:
This sounds like dead code. Why do we have a "framework"-specific test after killing developer/origin